package.json

```diff
@@ -363,6 +363,11 @@
           "default": null,
           "description": "Specifies the full path to the OmniSharp server."
         },
+        "omnisharp.experimentOmnisharp":{
+          "type":"string",
+          "default": null,
+          "description": "Specifies if the user wants to use the experiment build. Possible values - latest or some version number" 
```

Comment:
I think there's a way of specifying possible values so they show up in completions. Don't block on this, but something to consider.